### DISCUSSION TITLE
Will this work as an emulator

### DISCUSSION THREAD

**Discussion by raf802** [aligungr/UERANSIM#272] [Q&A] [answered]
Correct me if i'm wrong, but once the radio interface is developed, can this software be used to emulate a physical UE?
e.g. have a physical RF interface between the UE and gNB hosts?

Many thanks!

**Comment by aligungr** [comment, marked as answer]
Yes that's correct. However there are a huge amount of tasks to be done. Therefore radio interface will take a long time unless some volunteers in the community help us.

Implementing UE and gNB is probably the most difficult task in the entire field.

I believe that eventually I'll complete the all features in UE and gNB, but it'll take time.

**Reply by raf802** [reply]
I see, that is good news and I appreciate this is no small task!

Thank you for your reply.
